terminate called after throwing an instance of 'std::runtime_error'
  what():  File ../bmd-cuda/data/MNIST.in could not be opened!
